server/utils/auth/github_app_token.py

```
@@ -0,0 +1,356 @@
+
+    if response.status_code == 404:
+        raise GitHubAppInstallationNotFound(
+            f"GitHub installation_id={installation_id} not found (404)"
+        )
+
+    if response.status_code == 403:
+        body_text = response.text or ""
+        if "This installation has been suspended" in body_text:
+            raise GitHubAppInstallationSuspended(
+                f"GitHub installation_id={installation_id} is suspended"
+            )
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)